### PR TITLE
App-Bar Implementation

### PR DESCRIPTION
**Pull request by Jose-Hash-Dev**

### CODE REVIEW

**Comment by Jose-Hash-Dev** [comment]
nav

**Comment by Jose-Hash-Dev** [comment]
- History Button
- Logo (Temporary text)
- Profile Icon (Functionality will be added)
- Cart Icon opens details in a small menu - User can go to ProductDetail page or to Cart page
- Search Bar (Search functionality will be added)

File: components/navigation/icons/cart/style.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { Box, MenuItem } from "@mui/material";
+
+export const CartIconStyle = styled(LocalMallIcon)`
+  font-size: 22px;
+`;
+export const BoxContainer = styled(Box)`
+  display: flex;
+  align-items: center;
+  text-align: center;
+
```
Comment (Jalal999) [inline components/navigation/icons/cart/style.ts:12]: empty line?

File: components/navigation/style.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+export const Image = styled.img`
+  width: 100px;
+  height: 50px;
+`;
+
+export const ProfileText = styled(Typography)`
+  font-size: 16px;
+  font-family: Verdana,serif;
+  margin-right: 10px;
+
```
Comment (Jalal999) [inline components/navigation/style.ts:44]: ???

File: pages/cart/buttons/checkout/checkoutStyle.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+import { Button } from "@mui/material";
+
+export const CheckoutStyle = styled(Button)`
```
Comment (Jalal999) [inline pages/cart/buttons/checkout/checkoutStyle.ts:4]: do you use this style?

**Pull request merged**